conf/nginx/conf.d/referenceapi.server.conf

```diff
@@ -1,14 +1,14 @@
 server {
   listen      443 ssl;
 
-  server_name    api.chameleoncloud.org api.tacc.chameleoncloud.org;
+  server_name    api.dev.chameleoncloud.org;
```

Comment:
why is `dev` hardcoded here?

Reply:
You're right, this was left in by accident... Unfortunately, I'm not sure of the best way to switch on this value in nginx configuration. Trying to figure out a nice way to do this.